pom.xml

```diff
@@ -112,6 +112,7 @@
       <version.org.jacoco.plugin>0.7.9</version.org.jacoco.plugin>
 
       <owasp.version>1.4.3</owasp.version>
+      <spring.version>3.1.4.RELEASE</spring.version>
```

Comment:
Wow can we have a newer spring version, this is very old, they’re on 5.0.0 now

Reply:
Wow yes.

Reply:
I know, I agree this isn’t your change. But it’s still so very wow if this is new stuff going in for next release

Reply:
And yes I realise this should have been raised on original pr